pkg/cmd/container/create.go

```diff
@@ -123,6 +124,37 @@ func Create(ctx context.Context, client *containerd.Client, args []string, netMa
 	}
 	opts = append(opts, platformOpts...)
 
+	if imageRef, err := referenceutil.Parse(args[0]); err == nil && imageRef.Protocol == referenceutil.OCIArchiveProtocol {
+		// Load and create the platform specified by the user.
+		// If none specified, fallback to the default platform.
+		platform := []string{}
+		if options.Platform != "" {
+			platform = append(platform, options.Platform)
+		}
+
+		images, err := load.FromOCIArchive(ctx, client, imageRef.Path, types.ImageLoadOptions{
+			Stdout:       options.Stdout,
+			GOptions:     options.GOptions,
+			Platform:     platform,
+			AllPlatforms: false,
+			Quiet:        options.ImagePullOpt.Quiet,
+		})
+		if err != nil {
+			return nil, nil, err
+		} else if len(images) == 0 {
+			// This is a regression and should not occur.
+			return nil, nil, errors.New("OCI archive did not contain any images")
+		}
+
+		image := images[0].Name
+		// Multi-image archive provided, default to first image found.
+		if len(images) != 1 {
+			log.L.Warnf("multi-image OCI archive provided, defaulting to image %s...", image)
```

Comment:
```suggestion
			log.L.Warnf("multiple images are found for the platform, defaulting to image %s...", image)
```

Reply:
Applied in https://github.com/containerd/nerdctl/pull/3537/commits/932bfaf0a599fbda3f19377cc827092c371318bd